build/snapcraft.mk

```diff
@@ -0,0 +1,16 @@
+#
+# Makefile fragment for Snapcraft actions
+#
+SNAPCRAFT         ?= snapcraft
+
+snapcraft-login:
```

Comment:
Does this need to be called as a dep of `make release-publish`?

Reply:
Oh yes, thank you.